backoffice/src/Controllers/EmailController.cs

```diff
@@ -53,13 +53,20 @@ public async Task<IActionResult> SendEmail(Guid id)
         await _service.SendEmailAsync(id);
         
         //Add feedback (banner or something) to show sent. 
+        TempData["SentSuccess"] = "Email sent successfully";

```

Comment:
I would stick to maybe more generic keys like "SuccessMessage", "ErrorMessage" etc